package/com.unity.formats.usd/Samples/ExportMesh/Editor/Unity.Formats.USD.ExportMesh.Editor.asmdef

```diff
@@ -11,7 +12,9 @@
     "overrideReferences": false,
     "precompiledReferences": [],
     "autoReferenced": true,
-    "defineConstraints": [],
+    "defineConstraints": [
+        "UNITY_INCLUDE_TESTS"
```

Comment:
I'm not upto date on this one, so this is making the samples depend on the define for Tests being present while this is under Samples? (not sure if official docs but found this https://forum.unity.com/threads/info-on-unity_include_tests-define.890095/#post-5933573)

I see Slack indicates this PR fixes a compilation bug so I will check this out as expct helps explain this PR: https://unity.slack.com/archives/C04DABNK68J/p1686852220046129

Reply:
I _think_ this is making sure that the Samples assemblies aren't being included in a Player build. It seems to be unrelated to test code. It's needed here because it's set in the `Unity.Formats.USD.ExportMesh` assembly that this one depends on, and without it this assembly is being compiled into the player unnecessarily and thus having compile errors about missing the other assembly. Whether this is the 'right way', I'm not sure, it's just the way that previous devs have implemented it. I will have a play around and see if I can see what the reasoning is for it in the higher level asmdef, as I don't see it in any of the small sample of other package samples I've looked at.

Reply:
Concur with @vickycl. On of the types that are issue is"\Samples\ImportMesh\ImportMeshExample.cs" which isn't part of the editor, seems odd or maybe this is an issue arising from something 'legacy'

Reply:
I'm not really following what you mean there Craig- let's discuss in the Foundation sync later :) I have a feeling the correct fix would be to make all of the Samples asmdefs Editor-only, but I can't reproduce the error reliably so I haven't been able to test this and that it doesn't break anything else. I don't know if any of the test code reuses sample code either, which would not be possible in that case.

Reply:
Sounds like a plan. I'm no longer clear on what I stated as I see that 'UNITY_INCLUDE_TESTS' should effectively always be defined. However it appears this may not be the case the Package code so may depend how this is included into a project (TBC)